refact_webgui/webgui/tab_server_logs.py

```diff
@@ -48,7 +43,16 @@ async def get_log(right_file):
             return Response("File \"%s\" not found\n" % log_name, media_type="text/plain")
         right_file = list_of_files[-1]
 
+        async def streamer(last_n_lines: int = 10_000) -> AsyncIterator[str]:
+            for line in tailer.tail(open(right_file, 'r'), last_n_lines):
+                yield line + '\n'
+
+            if stream:
+                async for line in atail(right_file):  # (line, from_file)
```

Comment:
here we can loose some lines from working logs

Reply:
inthe case of streaming it should work exactly as
tail -n last_n_lines -f logfile